### ISSUE TITLE
Loss of course data when I edit a student

### ISSUE THREAD

**Issue by ralph1972** [juhahinkula/StudentListFinal#3]
Hi, I have a loss of course student data when I edit a student. It is normal? How can I solve it? Thanks

**Comment by juhahinkula** [comment]
No, that should not happen. I just run this locally and edition works fine without deletion.

**Comment by juhahinkula** [comment]
Thanks for reporting this bug. I'll fix it.

**Comment by ralph1972** [comment]
I double checked this issue: when I edit a student, the content of the student_course table for that student disappeard

**Comment by juhahinkula** [comment]
Yes, you are correct
